python_modules/dagster/dagster/core/definitions/sensor_definition.py

```diff
@@ -300,6 +308,12 @@ def minimum_interval_seconds(self) -> Optional[int]:
     def targets(self) -> Optional[List[Union[DirectTarget, RepoRelativeTarget]]]:
         return self._targets
 
+    @property
+    def jobs(self) -> Optional[Sequence[Union[GraphDefinition, JobDefinition]]]:
+        if len(self._targets) > 0 and isinstance(self._targets[0], DirectTarget):
+            return [t.pipeline for t in self._targets]
+        raise DagsterInvalidDefinitionError("No jobs were provided to SensorDefinition.")
```

Comment:
Do we want to allow this argument to be `jobs`, or just `job`? Do we anticipate continuing to support multiple jobs, or are we moving in the direction of one job per schedule/sensor?